### ISSUE TITLE
Implement NASA polynomials database builder and parser

### ISSUE THREAD

**Issue by xuwd11** [cs207group4/cs207-FinalProject#18]
1. Implement NASA polynomials database builder and parser (done; SQLBuilder.py and SQLParser.py in module).
2. Add documentation and tests.

**Comment by j6626** [comment]
Hi Jeffrey, 

I thought that we as a group were going to discuss how to split up the work appropriately.

**Comment by zeruniverse** [comment]
Hi Jane,

I didn’t do anything yet. I’m just making a note that we shouldn’t forget
it.

**Comment by zeruniverse** [comment]
Oh and the note is not under this issue. Then I don’t know why you
mentioned me specifically...

**Comment by j6626** [comment]
Sorry, I got those emails back to back and just clicked on one.

**Comment by xuwd11** [comment]
Hi all,

Sorry for the confusion. I happened to be in a symposium and had nothing else to do yesterday; so I worked on the project briefly before our meeting since I am not sure whether I have enough time to work on the project next week.

I guess we can move forward more efficiently as a group if someone has some initial trials, figures out the potential issues, and comes up with a concrete plan, which we can discuss upon, before we discuss how to split up the work.

Also, as the only group member from chemistry background, I assume it is reasonable for me to take care of the integration of reversible reactions (and NASA polynomials database parser has to be implemented before that), and make suggestions on how to make the software useful from a chemist's perspective. 

Again, sorry for the confusion I caused. We still have a lot of work to do since I just worked briefly and have done nothing more than some initial trials.

BTW, we can meet at/after the class tomorrow if we cannot finalize a time to meet today. (I am available all day today remotely though.)
